buildtools/webpack-migration

```diff
@@ -1,11 +1,21 @@
 #!/bin/bash -ex
 
+function jscodeshift {
+    if node_modules/.bin/jscodeshift --dry $*  | grep '^0 errors$'
```

Comment:
To speed up things you may want to use `tee` instead (untested):
```
node_modules/.bin/jscodeshift $* | tee jscodeshift_out.log
grep -q '^0 errors$'  jscodeshift_out.log && false
```

Reply:
Done :-)